src/logger.ts

```diff
@@ -90,39 +120,38 @@ export class Logger implements Disposable {
         if (this.fd || this.logToConsole) {
             const logMsg = `${getTimestamp()} ${scope}: ${msg}`;
             this.fd && fs.appendFileSync(this.fd, logMsg + "\n");
-            this.logToConsole && console[level == LogLevel.error ? "error" : "log"](logMsg);
+            this.logToConsole && console[level == vscode.LogLevel.Error ? "error" : "log"](logMsg);
         }
 
         // Half-baked attempt to avoid infinite loop.
         // Preferred approach is for modules to decide this via `createLogger(…, logToOutputChannel=…)`.
         const activeDoc = window.activeTextEditor?.document; // "output:asvetliakov.vscode-neovim.vscode-neovim"
         const outputFocused = activeDoc?.uri.scheme === "output" || activeDoc?.fileName?.startsWith("output:");
-
         if (logToOutputChannel && this.outputChannel && activeDoc && !outputFocused) {
             const fullMsg = `${scope}: ${msg}`;
             switch (level) {
-                case LogLevel.error:
+                case vscode.LogLevel.Error:
                     this.outputChannel.error(fullMsg);
                     break;
-                case LogLevel.warn:
+                case vscode.LogLevel.Warning:
                     this.outputChannel.warn(fullMsg);
                     break;
-                case LogLevel.info:
-                case LogLevel.debug:
-                    // XXX: `vscode.LogOutputChannel` loglevel is currently readonly:
-                    //      https://github.com/microsoft/vscode/issues/170450
-                    //      https://github.com/PowerShell/vscode-powershell/issues/4441
-                    // So debug() drops messages unless the user has increased vscode's log-level.
-                    // Use info() until vscode adds a way to set the loglevel.
```

Comment:
Does this problem no longer exist? When vscode starts, the log level is set to 'info'. It takes a while before `onDidChangeLogLevel` is triggered.

Reply:
This problem was resolved in recent versions of vscode. User can set the log-level on the channel. Note that they must also choose `Set as default`

<img width="321" alt="image" src="https://github.com/vscode-neovim/vscode-neovim/assets/1359421/fc563bfe-ecec-4a13-b03d-35d9540dc71b">